README.md

```diff
@@ -9,6 +9,10 @@ The leaderboard is website displays scores submitted by different players. It al
 - JavaScript (ES6 syntax and Modules)
 - Webpack
 
+## Deployment
+
+https://mohammedelseddik.github.io/Leaderboard/
+
 ## Getting Started
 
 To get a local copy up and running follow these simple example steps.
```

Comment:
[OPTIONAL] 

- [ ] Nice Job adding the steps to clone the repository locally. I kindly suggest that you also add steps to run the project locally.

### EXAMPLE:

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```

2. Start the development server

   ```sh
   npm start
   ```

3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.